modules/fts/pages/fts-creating-indexes.adoc

```diff
@@ -6,15 +6,25 @@ Full Text Searches are supported by specially purposed indexes, which can be cre
 [#indexes-and-full-text-search]
 == Indexes and Full Text Search
 
-Every Full Text Search is performed on a user-created _Full Text Index_, which contains the targets on which searches are to be performed: these targets are values derived from the textual and other contents of documents within a specified bucket.
+Every Full Text Search is performed on a user-created Full Text Index, which contains the targets on which searches are to be performed: these targets are values derived from the textual and other contents of documents within a specified bucket or collections within a scope.
 
-Index-creation is configurable, and can be highly selective: documents can be user-grouped into different _types_ (for example, `airline` documents versus `hotel` documents), based on the document-IDs or the values of a designated document-field; and each document-type can be assigned its own index-mapping.
-Each index-mapping in turn can be assigned its own _analyzers_, can be applied to a specific subset of document-fields, and can be explicitly included in or excluded from the index.
+Index-creation is configurable, and can be highly selective: documents can be user-grouped into different types (for example, airline documents versus hotel documents), different scope (for example, inventory) and collections (for example, airport or hotel). Based on the document-IDs or the values of a designated document-field; and each document-type can be assigned its own index-mapping.
```

Comment:
Must say that the description here could confuse a new user (and even existing users). Could do with better wording.

Reply:
Updated the text.